src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -219,7 +217,8 @@ private File getSaveFile() {
   private void saveFile(File fileToSave) {
     // set save file parent folder as the new default user folder
     FileUtils.setUserDirectory(fileToSave.getParent());
-    JsonParser.createFileFromRawContestConfig(fileToSave, createRawContestConfig());
+    // create a rawConfig object from GUI content and serialize it as json
+    JsonParser.parseObjectToFile(fileToSave, createRawContestConfig());
```

Comment:
This is just a semantic question, but is "parse" the right verb here? It feels like the opposite of what this is doing. "Serialize" would make more sense to me.

Reply:
yeah serialize is better

Reply:
agree, maybe I should change this to serialize / deserialize?